### PR TITLE
[4.3] Guided Tours disabled plugin

### PR DESCRIPTION
**Pull request by brianteeman**
### Summary of Changes
If the system guided tours plugin is disabled then the tours do not work

This simple pr adds a notice in the component if the plugin is disabled

### Testing Instructions

Disable the system plugin for guided tours
Open the guided tours component

### Actual result BEFORE applying this Pull Request
image

### Expected result AFTER applying this Pull Request
image

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by coolcat-creations** [comment]
I tested this successful but I am not sure about the message.

How about directing the user where and what to do?
So instead: The Guided Tours System Plugin is disabled. The Guided Tours module will not be displayed if this is not enabled.
Something like: Please enable the "The Guided Tours System Plugin" in System » Plugins for full functionality.

Why full functionality? If you say you have to enable the plugin to display the module, the user can be confused if the module is then still not displayed because the module could be unpublished too. Or we have to check for both to be enabled and published.

Hope it makes sense.

**Comment by brianteeman** [comment]
You are welcome to add additional changes

**Comment by coolcat-creations** [comment]
So what do you think, does it make sense? Is it correct english?

"The Guided Tours System Plugin is disabled. Please enable the “Guided Tours System Plugin” in System » Plugins for full functionality."

**Comment by brianteeman** [comment]
I would either keep it as is or enhance it by direcvtly linking to the plugin manager as is done with the redirect and smart search but the code in guidedtours was too different for me to do that.

File: administrator/components/com_guidedtours/src/Controller/DisplayController.php [hunk 1]
```diff
@@ -49,6 +50,11 @@ public function display($cachable = false, $urlparams = [])
         $layout = $this->input->get('layout', 'default');
         $id     = $this->input->getInt('id');
 
+        // Show messages about the disabled plugin
+        if ($view === 'tours' && !PluginHelper::isEnabled('system', 'guidedtours')) {;
```
Comment (richard67) [inline administrator/components/com_guidedtours/src/Controller/DisplayController.php]:
```suggestion
        if ($view === 'tours' && !PluginHelper::isEnabled('system', 'guidedtours')) {
```

**Comment by PjotrClaas** [comment]
I have tested this item :white_check_mark: successfully on 72c5e1f1a6f8821eda720e7ca31c16b9348960f6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40449">issues.joomla.org/tracker/joomla-cms/40449</a>.</sub>

**Comment by khu5h1** [comment]
I have tested this item :white_check_mark: successfully on 72c5e1f1a6f8821eda720e7ca31c16b9348960f6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40449">issues.joomla.org/tracker/joomla-cms/40449</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/40449">issues.joomla.org/tracker/joomla-cms/40449</a>.</sub>

**Comment by brianteeman** [comment]
Thanks

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**